app/assets/stylesheets/application.tailwind.css

```diff
@@ -147,4 +147,15 @@
   &:hover {
     @apply cursor-pointer text-secondary underline;
   }
+}
+.page-title{
```

Comment:
we have page-heading on line no 37, 
what is the difference between page-title and page-heading ?
 Since both kind of means the same how are we planning to distinguish both ? 
Do we need to make this a responsive class for mobile? 
Can we get rid of page-heading if page-title ?

Reply:
The page-title class was previously used in some course_module pages but didn’t have specific styling applied. I added Tailwind classes to page-title to meet the design requirements. The page-heading and page-title classes have distinct styling properties: page-heading is mainly used on login pages, while page-title is used in course_module contexts. Removing page-title class is possible

Reply:
both the names are similar so keeping one would be better if there is no other reason to keep both. If they have distinct styles can we name them distinctly ?